### ISSUE TITLE
A11y_Azure Extensions for VS Code_Functions (Disable Function)_Keyboard: Keyboard focus will get invisible after collapse configure notification button. Esc key for expand 

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode-azurefunctions#2225]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHub Tags:**#A11yTCS;#Azure Tools for VS Code;#A11y_Azure Extensions for VS Code; #DesktopApp;#Win10;;#MAS2.4.3;#Benchmark;#FTP;#A11yMAS;;#A11ySev2;#External: VSCode;

**Environment Details:**
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps:**

1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Click on"Local Project" from the treeview and expand it.
5. Click on the Functions menu and expand it further to see Triggers.
6. Right click on trigger to see the menu item "Disable Function"
7. Navigate to configure notification button and expand and then collapse with the help of esc key.

**Actual:**

Keyboard focus will get invisible after collapse configure notification button. Esc key for expand 

**Same issue is also observed with throughout the application.**

**Expected:**

Focus should be remain constant on the button from where we have started to perform action. Focus should move to "Configure notification button.

**User Impact:**

User will get confuse because of illogical focus order.

**Attachment:**
[#2225.zip](https://github.com/microsoft/vscode-azurefunctions/files/4889038/2225.zip)

**Comment by EricJizbaMSFT** [comment]
Filed the following issue on VS Code: https://github.com/microsoft/vscode/issues/102983

**Comment by harshgangrade111** [comment]
@EricJizbaMSFT Issue is repro in the application. Hence reactivating this bug. Please refer the attachment. After pressing "Esc" key keyboard focus outline should be visible back on Configure Notification button.
[#2225_Repro.zip](https://github.com/microsoft/vscode-azurefunctions/files/5223722/2225_Repro.zip)

**Comment by harshgangrade111** [comment]
As per the C & AI bug process assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.
[https://github.com/microsoft/vscode/issues/108296](https://github.com/microsoft/vscode/issues/108296)

**Comment by EricJizbaMSFT** [comment]
Closing since this bug is external and due to lack of response from accessibility test team on how to get this off our tracking list.
